client/js/app/controllers/NegociacaoController.js

```diff
@@ -15,7 +15,24 @@ class NegociacaoController {
         this._message = new Message();
         this._viewMessage = new MessageView($('#mensagem-view'));
         
-        this._listaNegociacoes = new ListaNegociacoes(model => this._viewNegociacoes.update(model));
+        let self = this;
```

Comment:
O this não pode ser utilizado dentro do escopo ao se referir a `_viewNegociacoes`, então criamos uma variável para fazer essa referência.